mercury/mercury-library/protocol-issue-credential/src/test/scala/io/iohk/atala/mercury/protocol/issuecredential/IssueCredentialSpec.scala

```diff
@@ -22,22 +22,19 @@ class IssueCredentialSpec extends ZSuite {
         .deepDropNullValues
         .noSpaces
 
-    val expectedProposalJson = parse(s"""{
-                         |    "id": "061bf917-2cbe-460b-8d12-b1a9609505c2",
-                         |    "type": "https://didcomm.org/issue-credential/2.0/issue-credential",
-                         |    "body":
-                         |    {
-                         |        "goal_code": "Issued Credential",
-                         |        "formats":
-                         |        []
-                         |    },
-                         |    "attachments":
-                         |    [
-                         |    $attachmentDescriptorJson
-                         |    ],
-                         |    "to" : "did:prism:test123",
-                         |    "from" : "did:prism:test123"
-                         |}""".stripMargin).getOrElse(Json.Null)
+      // FIXME !!! THIS WILL FAIL!
```

Comment:
Does it make sense to create a tech-debt for every FIXME entry?

Reply:
There is already a [tech debt ticket](https://input-output.atlassian.net/browse/ATL-3211) for it that was created some time ago. IMO, someone should go through all of those `FIXME` and `TODO` found in the codebase, then sort out/clean up and create specific tickets when deemed necessary.